web_widget_mail_send_odoo/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Web Widget - Email internal mail link
```

Comment:
reformulate to a sentence what this *does*, as in 'Use internal mail wizard for email links' or the like